app/code/Magento/Sales/Model/Order/Creditmemo/Total/Tax.php

```diff
@@ -95,6 +95,23 @@ public function collect(\Magento\Sales\Model\Order\Creditmemo $creditmemo)
                 $totalTax += $shippingTaxAmount;
                 $baseTotalTax += $baseShippingTaxAmount;
             }
+            
+            
+            $taxRefunded = 0;
+            $baseTaxRefunded = 0;
+            if ($order->hasCreditmemos()) {
+                
+                $invoiceCreditmemos = $order->getCreditmemosCollection()->addFieldToFilter('invoice_id', $invoice->getId());
+                
+                foreach ($invoiceCreditmemos as $invoiceCreditmemo) {
+                    $taxRefunded += $invoiceCreditmemo->getTaxAmount();
+                    $baseTaxRefunded += $invoiceCreditmemo->getBaseTaxAmount();
+                }
+            }
+            
+            $allowedTax = $invoice->getTaxAmount() - $taxRefunded - $creditmemo->getTaxAmount();
+            $allowedBaseTax = $invoice->getBaseTaxInvoiced() - $baseTaxRefunded - $creditmemo->getBaseTaxAmount();
```

Comment:
@nikoelgatito it should be `$invoice->getBaseTaxAmount()`

Reply:
@samicoman Thank you for pointing this error. I have submitted a fix through the commit #3ac0d99.
https://github.com/magento/magento2/pull/27301/commits/3ac0d9927f98520743baeed13d01d20021397f95